src/composables/useTokenGenerator/useTokenGenerator.test.ts

```diff
@@ -0,0 +1,26 @@
+import { describe, it, expect } from 'vitest';
+import useTokenGenerator from '@/composables/useTokenGenerator/useTokenGenerator';
+
+describe('useTokenGenerator Composable', () => {
+    const { generateToken, rules, minTokenLenght } = useTokenGenerator();
+
+    it('generateToken returns a string', () => {
+        const token = generateToken();
+        expect(typeof token).toBe('string');
+    });
+
+    it('generateToken generates a token of minimum length', () => {
+        const token = generateToken();
+        expect(token.length).toBeGreaterThanOrEqual(minTokenLenght.value);
+    });
+
+    it('generateToken generates a token that meets all rules', () => {
```

Comment:
It may be worth to test against a number of generated tokens (to exclude the possibilty that it was just an accident that the one token met all the criteria)

Reply:
I am not sure what you want now exacly

Reply:
We currently check whether one randomly generated token meets all the rules. Imo it would be better to check for like 10 generated tokens if they meet the rules

Reply:
what's the status on this? See no changes regarding that.